docs/index.md

```diff
@@ -52,6 +52,174 @@ A dictionary with a summary of the backup result.
 
 ---
 
+## cat.pack
+
+Output a pack object.
+
+### Args
+
+- `id_`: ID of the pack that's supposed to be printed.
+
+### Returns
+
+Pack object
+
+### Example
+
+```python
+>>> restic.cat.pack('5b10ede5b5819f0a9fff8b9f72869f51c5ee89b2d3dcc9c694958368cb3148a5')
+b'F\x03g\x9d\xea\xe0\xec\x0f\xb9\xba\x81\x19 \x18,lc\xec\x0b\x02\xc2\xc2\xe9,l\xcaXM\x[...]
```

Comment:
Does restic just ignore the `--json` flag for `restic cat pack`? If so, I still want resticpy to produce JSON for consistency with the rest of the APIs. Can we base64-encode the binary data and output it as a JSON object, like:

```python
>>> restic.cat.pack('5b10ede5b5819f0a9fff8b9f72869f51c5ee89b2d3dcc9c694958368cb3148a5')
{'data': 'bqi8Jk9quHtHAhLkDolasgtsMMx5EbfTWtalGSBDW65hSWm1Hpvg9KGmUuLdZryZnv1kts/XaG99+URGzaB8RQ84O0bMw=='}
```

Reply:
Yes, it's ignored and the pack is outputted as binary data. I'd rather not temper with the output of restic and just return what restic does. That's because the command return UX remains the same to restic.